<launch>
    <!--<node name="davinci_needle_pose_publisher" pkg="cwru_davinci_grasp" type="davinci_needle_pose_publisher" respawn="false" output="screen">-->
        <!--<rosparam command="load" file="$(find cwru_davinci_grasp)/config/grasping_and_needle_pose.yaml"/>-->
    <!--</node>-->
    # specify which arm is going to perform needle grasp
    <arg name = "which_arm" default = "psm_one"/>

    # specify the name of the needle
    <arg name = "needle_name" default = "needle_r"/>

    <node name="davinci_needle_grasper_action_server_main" pkg="cwru_davinci_grasp" type="davinci_needle_grasper_action_server_main" args="$(arg which_arm) $(arg needle_name)" respawn="false" output="screen">
        <rosparam command="load" file="$(find cwru_davinci_grasp)/config/dvrk_psm_grasp_needle_data.yaml"/>
    </node>
</launch>
